test/python/circuit/test_circuit_operations.py

```diff
@@ -799,6 +799,19 @@ def test_reverse_bits_with_registers(self):
 
         self.assertEqual(qc.reverse_bits(), expected)
 
+    def test_reverse_bits_isolated_qubits(self):
+        """Test reversing order of bits when QuantumCircuit is made with isolated Qubits (not in a register)."""
+        from qiskit.circuit import Qubit
```

Comment:
The normal reasons to protect an import (e.g. by putting it in a function) is because it can't be evaluated at the top of the module, or it requires optional dependencies.  In tests, it's quite unusual to need to do this, and it would be better just to put this import up at the top with the rest of them.